[3.2] docs(gantt): Revamped Overview and Gantt Splitter position para… (#880)
telerik/blazor-docs: components/gantt/gantt-tree/columns/command.md

* [Code Example](#example)

## Command Column Parameters

The Blazor Gantt Command Column provides various parameters to configure the component. Also check the [Gantt public API](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.Components.TelerikGantt-1).

### The GanttCommandButton Tag

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Parameter | Description |
| --- | --- |

> | `Command` | The command that will be invoked. Can be one of the built-in commands (see below), or a custom command name. |

Reviewer: "See below" can be a link?

```suggestion
| `Command` | The command to invoke. Can be one of the built-in commands (see below), or a custom command name. |
```

> | `ShowInEdit` | A parameter indicating whether the button is only visible while the user is editing/inserting data. |

Reviewer: ```suggestion
| `ShowInEdit` | Defines if the button is visible only when the user is editing or inserting data. |
```

> | `ChildContent` | The text the button will render. You can also place it between the command button's opening and closing tags. |

Reviewer: ```suggestion
| `ChildContent` | The text of the button. You can also place it between the command button's opening and closing tags. |
```
